src/json.ts

```diff
@@ -182,22 +182,36 @@ function validateJson(json: unknown): boolean {
   }
 
   if (typeof json === 'object') {
+    let every = true;
     if (Array.isArray(json)) {
-      return json.every((value) => validateJson(value));
+      // eslint-disable-next-line @typescript-eslint/prefer-for-of
```

Comment:
Can you add a comment explaining that for-of is significantly slower?

Reply:
Done!